lms/static/js/spec/search/search_spec.js

```diff
@@ -532,6 +533,7 @@ define([
             describe('DashSearchResultsView', function () {
                 beforeEach(function() {
                     beforeEachHelper.call(this, DashSearchResultsView);
+                    this.contentElementDisplayValue = 'block';
```

Comment:
Search views are used in two places: in courseware and in the dashboard. One the dashboard, we continue to use `block`. However, in the courseware, the course content wrapper now needs to have `display` value `table-cell` when it needs to be visbible.